workers/unity/Assets/Playground/Scripts/EntityInitialization/GameObjectInitializationSystem.cs

```diff
@@ -77,27 +84,15 @@ protected override void OnUpdate()
 
                 var position = new Vector3(transform.Location.X, transform.Location.Y, transform.Location.Z) +
                     worker.Origin;
-                var rotation = new UnityEngine.Quaternion(transform.Rotation.X, transform.Rotation.Y,
+                var rotation = new Quaternion(transform.Rotation.X, transform.Rotation.Y,
                     transform.Rotation.Z, transform.Rotation.W);
 
                 var gameObject =
                     entityGameObjectCreator.CreateEntityGameObject(entity, prefabName, position, rotation,
                         spatialEntityId);
-                var gameObjectReference = new GameObjectReference { GameObject = gameObject };
-
-                var requiresSpatialOSBehaviourManagerComponent = new RequiresMonoBehaviourActivationManager();
-
+                entityGameObjectLinker.LinkGameObjectToEntity(gameObject, entity, spatialEntityId, viewCommandBuffer);
                 entityGameObjectCache.Add(entity, gameObject);
-                var gameObjectReferenceHandleComponent = new GameObjectReferenceHandle();
-
-                PostUpdateCommands.AddComponent(addedEntitiesData.Entities[i], gameObjectReferenceHandleComponent);
-
-                PostUpdateCommands.AddComponent(addedEntitiesData.Entities[i],
-                    requiresSpatialOSBehaviourManagerComponent);
-
-                viewCommandBuffer.AddComponent(entity, gameObjectReference);
-                entityGameObjectLinker.LinkGameObjectToEntity(gameObject, entity, spatialEntityId,
-                    viewCommandBuffer);
+                PostUpdateCommands.AddComponent(entity, new EntityGameObjectHandle());
```

Comment:
Is there not a `PostUpdateCommands.AddComponent<EntityGameObjectHandle>(entity)`?

Reply:
nope, there is not. it's very troll